e2e/benchmarks/local-benchmark/index.html

```diff
@@ -243,8 +221,9 @@ <h2>TensorFlow.js Model Benchmark</h2>
             console.warn(e.message);
           }
 
-          const debug = keepIntermediateTensors & (benchmarks[state.benchmark].supportDebug !== false);
-          referenceData = await predictAndGetPredictionData(predict, model, inferenceInput, debug);
+          const enableDump = keepIntermediateTensors & (benchmarks[state.benchmark].supportDump !== false);
+          const referenceResult = await predictAndGetData(predict, model, inferenceInput, state.benchmark, enableDump);
```

Comment:
Can we unify the terms here? Use "expected" for reference, and use "actual" for actual results

Reply:
done